packages/wdio-browserstack-service/src/ai-handler.ts

```diff
@@ -0,0 +1,212 @@
+        const sessionId = browser.sessionId
+
+        let tcgDetails = `{"region": "${TCG_INFO.tcgRegion}", "tcgUrls": {"${TCG_INFO.tcgRegion}": {"endpoint": "${TCG_INFO.tcgUrl.split('s://')[1]}"}}}`
+        tcgDetails = tcgDetails.replace(/'/g, '\\\'').replace(/"/g, '\\"')
+        const locatorType = using.replace(/'/g, '\\\'').replace(/"/g, '\\"')
+        const locatorValue = value.replace(/'/g, '\\\'').replace(/"/g, '\\"')
+
+        this.authResult = this.authResult as BrowserstackHealing.InitSuccessResponse
+
+        try {
+            const result = await orginalFunc(using, value)
+            if (!result.error) {
+                const script = await aiSDK.BrowserstackHealing.logData(locatorType, locatorValue, undefined, undefined, this.authResult.groupId, sessionId, undefined, tcgDetails)
+                if (script) {
+                    await browser.execute(script)
+                }
+                return result
+            }
+            if (options.selfHeal === true && this.authResult.isHealingEnabled) {
+                BStackLogger.info('findElement failed, trying to heal')
+                const script = await aiSDK.BrowserstackHealing.healFailure(locatorType, locatorValue, undefined, undefined, this.authResult.userId, this.authResult.groupId, sessionId, undefined, undefined, this.authResult.isGroupAIEnabled, tcgDetails)
+                if (script) {
+                    await browser.execute(script)
+                    const tcgData = await aiSDK.BrowserstackHealing.pollResult(TCG_URL, sessionId, this.authResult.sessionToken)
+                    if (tcgData && tcgData.selector && tcgData.value){
+                        const healedResult = await orginalFunc(tcgData.selector, tcgData.value)
+                        BStackLogger.info('Healing worked, element found: ' + tcgData.selector + ': ' + tcgData.value)
+                        return healedResult.error ? result : healedResult
+                    }
+                }
+            }
+        } catch (err) {
+            BStackLogger.debug('Error in findElement: ' + err + 'using: ' + using + 'value: ' + value)
+            if (options.selfHeal === true) {
+                BStackLogger.debug('Something went wrong while healing. Disabling healing for this command')
+            }
+        }
+        return await orginalFunc(using, value)
+    }
+
+    addMultiRemoteCaps (
+        authResult: BrowserstackHealing.InitSuccessResponse | BrowserstackHealing.InitErrorResponse,
+        config: Options.Testrunner,
+        browserStackConfig: BrowserStackConfig,
+        options: BrowserstackOptions,
+        caps: any,
+        browser: string
+    ) {
+        if ( caps[browser].capabilities &&
+            !(isBrowserstackInfra(caps[browser])) &&
+            SUPPORTED_BROWSERS_FOR_AI.includes(caps[browser].capabilities.browserName)
```

Comment:
handle cases and other names like?
chrome, Chrome, edge, Edge...

Reply:
Added a `.toLowerCase()` function

Reply:
Ticket to support edge: https://browserstack.atlassian.net/browse/SDK-1230